agusit7/lipizzaner-gan
# Migrate diverse lipi to MPI

To test locally run:
`mpiexec -n 4 python src/main_mpi.py src/configuration/lipizzaner-gan/mnist.yml`
To run on cluster:
`sbatch lipizzaner-gan/launch_mpi.sh`
Missing:
- Adapt some of the methods with the latest MOO implementations
- Check the implementation of EGAN loss
